libraries/rush-lib/src/logic/operations/OperationExecutionManager.ts

```diff
@@ -255,7 +258,8 @@ export class OperationExecutionManager {
       async (record: OperationExecutionRecord) => {
         await record.executeAsync({
           onStart: onOperationStartAsync,
-          onResult: onOperationCompleteAsync
+          beforeResult: beforeOperationResult,
+          onResult: this._onOperationComplete.bind(this)
```

Comment:
Don't rebind this on every call, just do so once outside of `Async.forEachAsync`.